resources/upgrading-to-aep/README.md

```diff
@@ -40,6 +40,7 @@ Review the following checklist to understand some of the changes and what is req
    **Tip**: The Experience Platform SDK supports iOS versions 10+, Android 4+ \(API 14+\).
 
 3. If you are implementing Analytics, see [Processing rules overview](https://docs.adobe.com/content/help/en/analytics/admin/admin-tools/processing-rules/processing-rules.html) to map the variables and rules.
+4. In the Experience Platform SDK, you will need to manually implement your lifecycle calls. See [Manual Lifecycle Implementation](manual-lifecycle-implementation.md) for more details.
```

Comment:
Please reword as follows:

"In the Experience Platform SDK, you need to manually implement your lifecycle calls. For more information, see [Manual Lifecycle Implementation](manual-lifecycle-implementation.md).